tests/Doctrine/DBAL/Migrations/Tests/Finder/_files/_symlinked_files

```diff
@@ -0,0 +1 @@
+../_symlinked_files/
```

Comment:
Will this work on Windows? Maybe _skip-if_ should be added for that poor thing? :)

Reply:
Probably not, but then our test suite doesn't run on Windows right now and I'd expect the directory iterators to know how to handle Symlinks on Windows ;)